src/index.js

```diff
@@ -372,6 +450,44 @@ class EventBridgeHttpCollector extends Collector {
   }
 }
 
+
+class CodeCommitHttpCollector extends Collector {
+  constructor(lintEnv) {
+    super('simple', lintEnv);
+  }
+
+  /**
+   * This code process CodeCommit Logs
+   * @param {*} logsJson 
+   */
+  processLogsJson(logsJson) {
+    if (!logsJson.Records) {
+      throw new Error('JSON blob does not have log records. Skip processing the blob.');
+    }
+
+    processCodeCommitLogs(logsJson.Records);
+    return JSON.stringify(logsJson.Records);
+  }
+}
+
+class MSKHttpCollector extends Collector {
+  constructor(lintEnv) {
+    super('simple', lintEnv);
+  }
+
+  /**
+   * This code process MSK Logs
+   * @param {*} logsJson 
+   */
+  processLogsJson(logsJson) {
+    processMSKLogs(logsJson);
+    if (!logsJson.Records) {
```

Comment:
this will fail i think.

It should be like this ---- small case records as per your test data test/sample_files/msk_testdata.js
if (!logsJson.records) {